package.json

```diff
@@ -32,9 +32,9 @@
     "release": "ts-node-script scripts/release/cli.ts",
     "pretest": "node tools/pretest.js",
     "test": "lerna run --concurrency 4 --stream test",
-    "test:ci": "lerna run --concurrency 4 --stream test:ci",
-    "test:release": "lerna run --concurrency 4 --ignore @firebase/*-exp --ignore firebase-exp --stream test:ci",
-    "test:exp": "lerna run --concurrency 4 --stream --scope @firebase/*-exp --scope firebase-exp test",
+    "test:ci": "lerna run --concurrency 4 test:ci",
+    "test:release": "lerna run --concurrency 4 --ignore @firebase/*-exp --ignore firebase-exp test:ci",
+    "test:exp": "lerna run --concurrency 4 --scope @firebase/*-exp --scope firebase-exp test",
```

Comment:
test:exp falls into the same category I think? It's not a CI command, right?

Reply:
yup, added it back